dom-creation/main.js

```diff
@@ -0,0 +1,96 @@
+    description: 'It has a preference for hot things. When it rains, steam is said to spout from the tip of its tail.',
+    imageUrl: 'images/charmander.png'
+  },
+  {
+    number: '007',
+    name: 'Squirtle',
+    description: 'When it retracts its long neck into its shell, it squirts out water with vigorous force.',
+    imageUrl: 'images/squirtle.png'
+  },
+  {
+    number: '002',
+    name: 'Ivysaur',
+    description: 'When the bulb on its back grows large, it appears to lose the ability to stand on its hind legs.',
+    imageUrl: 'images/ivysaur.png'
+  },
+  {
+    number: '005',
+    name: 'Charmeleon',
+    description: 'It has a barbaric nature. In battle, it whips its fiery tail around and slashes away with sharp claws.',
+    imageUrl: 'images/charmeleon.png'
+  },
+  {
+    number: '008',
+    name: 'Wartortle',
+    description: 'It is recognized as a symbol of longevity. If its shell has algae on it, that Wartortle is very old.',
+    imageUrl: 'images/wartortle.png'
+  },
+  {
+    number: '003',
+    name: 'Venusaur',
+    description: 'Its plant blooms when it is absorbing solar energy. It stays on the move to seek sunlight.',
+    imageUrl: 'images/venusaur.png'
+  },
+  {
+    number: '006',
+    name: 'Charizard',
+    description: 'It spits fire that is hot enough to melt boulders. It may cause forest fires by blowing flames.',
+    imageUrl: 'images/charizard.png'
+  },
+  {
+    number: '009',
+    name: 'Blastoise',
+    description: 'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png'
+  }
+];
+
+function renderPokemon(pokemon) {
+
+  var columnDiv = document.createElement('div');
+  rowDiv.appendChild(columnDiv);
+  columnDiv.setAttribute('class', 'column-third');
+
+  var pokemonCardDiv = document.createElement('div');
+  columnDiv.appendChild(pokemonCardDiv);
+  pokemonCardDiv.setAttribute('class', 'pokemon-card');
+
+  var pokemonImage = document.createElement('img');
+  pokemonImage.setAttribute('src', pokemon.imageUrl);
+  pokemonImage.setAttribute('alt', pokemon + ' Picture');
+  pokemonCardDiv.appendChild(pokemonImage);
+
+  var pokemonCardText = document.createElement('div');
+  pokemonCardText.setAttribute('class', 'pokemon-card-text');
+  pokemonCardDiv.appendChild(pokemonCardText);
+
+  var pokemonH2 = document.createElement('h2');
+  pokemonH2.textContent = pokemon.name;
+  pokemonCardText.appendChild(pokemonH2);
+
+  var pokemonH3 = document.createElement('h3');
+  pokemonH3.textContent = pokemon.number;
+  pokemonCardText.appendChild(pokemonH3);
+
+  var pokemonP = document.createElement('p');
+  pokemonP.textContent = pokemon.description;
+  pokemonCardText.appendChild(pokemonP);
+}
+var rowDiv = document.querySelector('div.row');
+for (var i = 0; i < pokedex.length; i++) {
+  renderPokemon(pokedex[i]);
+  var allTogetherNow = document.createElement('node');
+  rowDiv.appendChild(allTogetherNow);
```

Comment:
I'm not sure what the intended purpose of these lines was, but they can be removed.